gravitee-apim-console-webui/src/management/api/endpoints-v4/endpoint/api-endpoint.component.ts

```diff
@@ -216,6 +255,53 @@ export class ApiEndpointComponent implements OnInit, OnDestroy {
       if (!inheritConfiguration) {
         sharedConfigurationOverride = this.endpoint.sharedConfigurationOverride;
       }
+      healthCheckInherited = !this.endpoint.services?.healthCheck?.overrideConfiguration;
```

Comment:
```suggestion
      healthCheckInherited = this.endpoint.services?.healthCheck?.overrideConfiguration != null;
```

I've first missed the `!` thinking we were storing the overridden config in that variable. The suggestion I've provided makes it explicit we are checking if an override exists

Reply:
I think the `!` must be there because `inherited` logically is opposite to the `override`. If you override it means it's not inherited from parent. 
We already check if override exists with `?`, are we?

Reply:
my bad, the condition should be 

```
this.endpoint.services?.healthCheck?.overrideConfiguration == null;
```

It was a suggestion because the when reading the code, I missed the `!` therefore I thought we were only storing the overriden configuration and I've started to write a comment saying that the name of the variable were not great. But when I read the following code, I noticed that the `healthCheckInherited` was a boolean and I saw the `!`.

The suggestion I make should help future reader to quickly know that `healthCheckInherited` is a boolean 🤷

Reply:
Maybe @ytvnr or @Okhelifi can give their opinion but I'm ok to leave it that way ;)

Reply:
no problem with the `!` imho, but indeed it is not clear at first glance that `healthCheckInherited` is a boolean
In this kind of case, I try to use a verb in a "getter style" as it is done here: https://github.com/gravitee-io/gravitee-api-management/blob/6cc64fc95da24eca5a84fe4bc10771650a181ca7/gravitee-apim-console-webui/src/management/api/api-navigation/api-v4-menu.service.ts#L39

So the suggestion would be to call it `isHealthCheckInherited` to know directly it is a boolean

Reply:
I agree. Renaming the variable also helps 🙈

Reply:
makes sense for me, renamed, thanks guys! @jgiovaresco would you reapprove if all good?